_pkgdown.yml

```diff
@@ -0,0 +1,21 @@
+url: https://dfe-analytical-services.github.io/dfeshiny/
+template:
+  bootstrap: 5
+  bootswatch: cyborg
```

Comment:
Distinctly less jazzy than the rainbow you could have had...

In all seriousness it works well / looks good, aside from I think we need to add some custom CSS for the selected state in the navbar as it's pretty hard to read
![image](https://github.com/dfe-analytical-services/dfeshiny/assets/52536248/c1c1effa-5f9e-4f39-bf54-5e9ba3d98c4a)

Reply:
Fixed. Gone for some nice blues!

Reply:
So you have - they work very nicely with the logo 👏